### ISSUE TITLE
[Feature Request] Add possibility to rename game folders to their IGDB name

### ISSUE THREAD

**Issue by shawly** [grimsi/gameyfin#65]
# Add rename button for games

It would be great to have the possibility to rename my game folders and files to the IGDB name.

## Proposed solution

Add a rename button in the actions column of the "Game mappings" table and for the "Game mapping" dialog with an extra button "Rename" so unmapped files can be directly renamed and mapped.

Pressing the rename button, Gameyfin should rename the game folder if it is a directory or the file (preserving the extension of course).
This should ideally store the old path in a `history` table so it doesn't get lost if users want to revert the rename action.

## Examples:
![image](https://user-images.githubusercontent.com/6448794/197387073-66b41ae6-bbd4-4094-88ed-16b285326ef8.png)

![image](https://user-images.githubusercontent.com/6448794/197386895-aab93bd1-36d9-4ba1-8ea4-8c065160e665.png)

**Comment by grimsi** [comment]
Yup, looks like a handy feature. But will be pretty low on the priority list since it's more of a convenience than a "must-have"

**Comment by shawly** [comment]
Of course I'll volunteer to implement the feature requests I open. Just thought it would be better to keep track of things, so nobody else will work on the same stuff! ☺️

**Comment by dsbeans** [comment]
Sorry to jump in on this thread, but I'd just like to add that I think that in addition to renaming to the slug, I think it would be very useful for game identification if the system would use the folder name as the igdb slug if it is in kebab case. I'm having some difficulty getting matches, especially for a lot of classic games from the 80s and early 90s, so I've just opted to find the slug off of igdb and putting that as the folder name myself - having it recognize that it is a slug and use that by default would be great.

**Comment by freitagdavid** [comment]
If this feature was added this would cover precisely the use case I'm wanting. Preferably you could give it some sort of renaming string like `%platform%/%title% (%year%)`, something along those lines would be great.
